tests/data/placebo/test_rdsclusterparamgroup_filter/rds.DescribeDBClusterParameters_1.json

```diff
@@ -0,0 +1,1321 @@
+{
```

Comment:
lets trim down the recorded test data to a single file

Reply:
@thisisshi not sure how to trim test data and document I can refer ?

Reply:
@RazaHasan84 my understanding is that @thisisshi wants you to manually modify the recorded test data files. There are hundreds of parameters that we simply don't care about and we can remove most of them. Keep the parameters that you're testing for in the filter, plus maybe additional 3-5 parameters. The tricky part though is I don't know if we can just simply merge everything into 1 single file. I believe we would need 2 because there are 2 DB Clusters so at the minimum, there needs to be 2 calls and responses for "describe db cluster parameters". Right @thisisshi ?

Reply:
right, we can reduce it down to a single file thanks @darrendao

Reply:
@darrendao @thisisshi trimmed the test data